package.json

```diff
@@ -88,9 +90,11 @@
     "streamify-array": "^1.0.1",
     "ts-jest": "^29.0.5",
     "ts-loader": "^9.4.1",
+    "ts-node": "^10.9.2",
```

Comment:
Not necessary?

Reply:
It was needed for either Webpack or Jest to be able to parse the TS config files.

Reply:
This then ties into my other comment on if we really need to migrate all these js config files to ts. So let's continue this discussion there.